src/Modal/Modal.stories.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import PropTypes from 'prop-types';
+
+import Modal from './index';
+import Button from '../Button';
+
+class ModalWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.openModal = this.openModal.bind(this);
+    this.resetModalWrapperState = this.resetModalWrapperState.bind(this);
+
+    this.state = { open: false };
+  }
+
+  openModal() {
+    this.setState({
+      open: true,
+    });
+  }
+
+  resetModalWrapperState() {
+    this.setState({ open: false });
+  }
+
+  render() {
+    return (
+      <div>
+        <Modal
+          open={this.state.open}
+          title={this.props.title}
+          body={this.props.body}
+          onClose={this.resetModalWrapperState}
+        />
+        <Button
+          onClick={this.openModal}
+          display="Click me to open a modal!"
+          buttonType="light"
+        />
+      </div>
+    );
+  }
+}
+
+ModalWrapper.propTypes = {
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
+  body: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
+};
+
+ModalWrapper.defaultProps = {
+  open: false,
+};
+
+storiesOf('Modal', module)
+  .add('basic usage', () => (
+    <Modal
+      open
+      title="Modal title."
+      body="Modal body."
+    />
+  ))
+  .add('configurable buttons', () => (
+    <Modal
+      open
+      title="Modal title."
+      body="Modal body."
+      buttons={[
+        <Button
+          display="Blue button!"
+          buttonType="primary"
+        />,
+        {
+          display: 'Red button!',
+          buttonType: 'danger',
+        },
+        <Button
+          display="Green button!"
+          buttonType="success"
+        />,
+      ]}
+    />
+  ))
+  .add('configurable title and body', () => (
+    <Modal
+      open
+      title="Custom title!"
+      body="Custom body!"
+      buttons={[
+        <Button
+          display="Dark button!"
+          buttonType="dark"
+        />,
+      ]}
+    />
+  ))
+  .add('configurable buttons that perform actions', () => (
+    <Modal
+      open
+      title="Modal title."
+      body="Modal body."
+      buttons={[
+        <Button
+          display="Click me and check the console!"
+          buttonType="light"
+          onClick={() => console.log('I made a log!')}
+        />,
+      ]}
+    />
+  ))
+  .add('modal invoked via a button', () => (
+    <ModalWrapper
+      title="I am the modal!"
+      body="I was invoked by a button!"
+    />
+  ));
```

Comment:
That's a good idea! Do you think adding, say, a button to the above example that would, theoretically, send you an e-mail with a book (theoretically, as I have no book to send), would be useful? I'd, of course, create some kind of story in which a button would actually be functional and wouldn't required sending e-mail. It would require another wrapper class, though, akin to ModalWrapper, and I fear that may be too complicated for the purposes of the storybook. I was thinking something along the lines of... select your favorite month, click a button, and I'll tell you what season it's in (maybe with a coordinating emoji), or something silly like that. Either way is fine with me!